.circleci/config.yml

```diff
@@ -134,7 +134,6 @@ jobs: # a collection of steps
           command: |
             yarn sample prettify
             yarn sample lint
-            yarn sample flow
```

Comment:
That's one solution 😆 

I think you can add the custom flow typing at `js-miniapp-sample/flow-typed/npm/@brightclove/react-player-loader_vx.x.x.js`, and then you just have to define this content in the file:

```
declare module '@brightcove/react-player-loader' {
    declare module.exports: any;
}
```

Actually, maybe we can convert this project to TypeScript sometime in the future? It would make sense because the other projects in the repo are using TypeScript.